### PR TITLE
2023!

### PR DESCRIPTION
**Pull request by RaptaG**

### CODE REVIEW

**Review by osfanbuff63 — COMMENTED** [review]
Never knew how this worked that much, but is 2022-2023 right or just 2023? /shrug

File: LICENSE.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ Redistribution and use in source and binary forms, with or without modification,
...
\ No newline at end of file
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (Madis0) [inline LICENSE.md:11]: What has changed here?

Comment (osfanbuff63) [reply LICENSE.md:11]: spacing maybe? not sure

**Review by Kichura — APPROVED** [review]
~~Personal opinion: "2020-2023" would've been cooler although this is good too.~~ actually no it's not quite.

File: LICENSE.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright 2022 Fabulously Optimized Authors
+Copyright 2023 Fabulously Optimized Authors
```
Comment (Kichura) [inline LICENSE.md:1]: `Copyright 2020-2023 Fabulously Optimized Authors`

this makes morely sense in practice.

**Review by Kichura — CHANGES_REQUESTED** [review]
was wrong about my approval, oop-

File: LICENSE.md (hunk 1 above)
Comment (RaptaG) [reply LICENSE.md:11]: I did nothing here, just a casual GitHub moment

**Pull request merged**